Appraisals

```diff
@@ -1,32 +1,38 @@
 appraise 'rails-6-0' do
   gem 'rails', '6.0.6.1'
+  gem 'activerecord-trilogy-adapter', '>= 3.1.2'
```

Comment:
Why not put this in the gemspec?

Reply:
@benlangfeld like mysql2, trilogy is also an optional dependency. Also, we only need this gem on rails < 7.1. This is shipped with rails on 7.1.

Reply:
If someone is using Trilogy, they would likely not want to have to install mysql2, correct? Should we not remove both from the gemspec?

Reply:
@benlangfeld mysql2 is listed as dev dependency, so it wouldn't be installed. Ideally, trilogy would be there too, but you _cannot_ have the adapter installed when you have rails 7.1, and thus relying on Appraisal is a better approach.

Reply:
#107 is the goal. We switched from not pushing our lock files to pushing them now, I'll soon take time to set up renovate as well.